Ubuntu 22 04 (#1281)
theforeman/foreman-documentation: guides/common/modules/proc_provisioning-ubuntu-20-04-3-plus-through-smart-proxies.adoc

[id="Provisioning_Ubuntu_20_04_3_plus_Through_Smart_Proxies_{context}"]

> = Provisioning Ubuntu 20.04.3+ Through {SmartProxies}

Author: Do you like this title @bastian-src ? Should it rather include "Ubuntu Autoinstall"?

Reviewer: Yes, we should say:
```suggestion
= Provisioning Ubuntu Autoinstall Through {SmartProxies}
```
Since you mention the specific versions in the short notice beneath, we don't need the version here.

Author: Applied.

Perform these steps to provision hosts running Ubuntu 20.04.3+ or Ubuntu 22.04.

.Procedure
. On your {SmartProxyServer}, ensure _Smart Proxy HTTP forward_ is enabled:
+
[options="nowrap", subs="+quotes,verbatim,attributes"]
----
# grep "^:enabled: true$" /etc/foreman-proxy/settings.d/templates.yml
----
+
Note that this is enabled by default.

> . On your {SmartProxyServer}, open the corresponding port for the Smart Proxy service:

Author: I am unsure if we can/should say "Smart Proxy service" here.

Reviewer: I get what you mean, service is a term which makes me think more about a daemon and that wouldn't fit too well here.
Instead, we could say:
```suggestion
. On your {SmartProxyServer}, open the corresponding port so the Smart Proxy can process incoming requests.
```

Author: Removed "for the Smart Proxy service" instead.
